Classes/git/PBGitGrapher.mm

```diff
@@ -46,22 +46,33 @@ - (id) initWithRepository: (PBGitRepository*) repo
 	return self;
 }
 
-void add_line(struct PBGitGraphLine *lines, int *nLines, int upper, int from, int to, int index)
+int long_to_integer_heuristic(long input)
+{
+	if (input < INT_MIN) {
+		return INT_MIN;
+	} else if (input > INT_MAX) {
+		return INT_MAX;
+	}
+	return (int)input;
+}
```

Comment:
Thanks for the explanations @nanotech : I inserted the function from your example code now.